h2o-py/tests/testdir_algos/gbm/pyunit_PUBDEV_4090_grid_model_seeds.py

```diff
@@ -81,8 +81,8 @@ def random_grid_model_seeds_PUBDEV_4090():
                                                            "got %s" % (expectedSeeds, model2Seeds)
 
     # compare training_rmse from scoring history
-    metric_list1 = pyunit_utils.extract_scoring_history_field(air_grid1.models[0], "training_rmse")
-    metric_list2 = pyunit_utils.extract_scoring_history_field(air_grid2.models[0], "training_rmse")
+    metric_list1 = pyunit_utils.extract_scoring_history_field(air_grid1.models[0], "training_rmse", True)
```

Comment:
Does this change have anything to do with the subject of the PR "PUBDEV-5072: merge duplicate"?

Reply:
Nope

Reply:
It fixes an intermittent.  I can do a separate PR for it if you want.